features/projects/components/project-error/project-error.tsx

```diff
@@ -17,7 +17,8 @@ const Container = styled.div`
   ${textFont("sm", "medium")};
 `;
 
-const ErrorButton = styled(Button)`
+const ErrorButton = styled.button`
```

Comment:
I'm wondering if it would make sense to add another button version to the `Button` component. A button that looks like a text link basically. But this approach is fine as well. At least the CSS reset is reused 👍

Reply:
I figured that is the only other logical solution. I tried using the new button component for these existing buttons then quickly tried something else since it got real messy real quick...